src/Components/About.jsx

```diff
@@ -1,37 +1,21 @@
 import React from "react";
 import "./../Styles/index.css"
-import Ava from './../Image/ava.jpg';
-import Css from './../Image/CSS.png';
-import Git from './../Image/GIT.png';
-import Html from './../Image/HTML.png';
-import Js from './../Image/JS.png';
-import Next from './../Image/NEXT.png';
-import Reactp from './../Image/REACT.png';
-
+import Ava from './../Image/alpha.jpg';
 
 export const About = () => {
     return(
-        <div className="fade">
-            <div className="about">
+        <div className="ret">
+            <div className="about flex">
                 <div>
-                    <img className="ava" src={Ava} alt="ava" />
+                    <img className="ava" src={Ava} alt="" />
```

Comment:
I try to make a habit of asking alt title, it's a useful skill that will help you at work.

Reply:
Read about semantics